tests/Database/Oci8SchemaGrammarTest.php

```diff
@@ -686,9 +699,8 @@ public function testAddingIncrementingID()
         $blueprint->increments('id');
         $statements = $blueprint->toSql($this->getConnection(), $this->getGrammar());
 
-        $this->assertEquals(1, count($statements));
-        $this->assertEquals('alter table "USERS" add ( "ID" number(10,0) not null, constraint users_id_pk primary key ( "ID" ) )',
```

Comment:
I need to review this further, the primary key constraint were removed on several tests.